voltixApp/VoltixApp/Chains/THORChainSwaps.swift

```diff
@@ -120,6 +125,9 @@ class THORChainSwaps {
             case .eth:
                 // TODO what if it is ERC20
                 return EVMHelper.getEthereumHelper().getSignedTransaction(vaultHexPubKey: self.vaultHexPublicKey, vaultHexChainCode: self.vaultHexChainCode, inputData: inputData, signatures: signatures)
```

Comment:
Here we will need to consider ERC20 tokens and BRC20 tokens , but you can use a seperate PR to fix it , given this one is pretty big already